packages/edge-demo-site/app/routes/index.tsx

```diff
@@ -1,3 +1,9 @@
+export function headers({ loaderHeaders, parentHeaders }: { loaderHeaders: Headers; parentHeaders: Headers }) {
+  return {
```

Comment:
Put this in as an example because cache headers in the netlify.toml weren't being picked up by edge functions.

Reply:
Even as an example, I'd suggest not using such a long cache expiry. It could lead to unexpected problems.